.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Description
+Provide a brief overview of the enhancement or issue.
+
+## Steps to Reproduce
+Be as detailed as possible when providing steps to reproduce. 
```

Comment:
Let's reword this to `Please add as much detail as possible to reproduce the error/issue`